src/MeetingsSDKAdapter.js

```diff
@@ -1038,6 +1039,18 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Switches the layout control.
+   *
+   * @param {string} ID  Meeting ID
+   * @param {string} layoutType  Type of layout
+   */
+  async changeLayout(ID, layoutType) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting.changeVideoLayout(layoutType);
```

Comment:
in order to wait for the result of changeVideoLayout, and return the error (if any):
```js
await sdkMeeting.changeVideoLayout(layoutType);
```
or remove the `async` from the function and:
```js
return sdkMeeting.changeVideoLayout(layoutType);
```

Also, add `@async` in jsdoc and `@returns {Promise}`